src/app/Http/Controllers/API/PollController.php

```diff
@@ -38,7 +39,18 @@ public function index(Presentation $presentation)
      */
     public function store(Request $request, Presentation $presentation)
     {
-        //
+        $request->validate([
+            'subject' => 'required',
+            'status' => 'required'
```

Comment:
status should not be passed client side,
this is part of internal logic the user should not have access to modify those parameters
cf: API documentation!

Reply:
Forgot to remove superfluous validation as per overwriting assignment in following lines, fixed in b75458f